packages/nextjs/hooks/scaffold-eth/useWalletBalance.ts

```diff
@@ -0,0 +1,30 @@
+import { useCallback, useEffect, useState } from "react";
+import { useBalance } from "wagmi";
+import { useAppStore } from "~~/services/store/store";
+
+export function useWalletBalance(address?: string) {
```

Comment:
I like the idea of extracting this functionality as a hook!

The name... what about `useAddressBalance`, or `useAccountBalance`? Seems more appropriate than Wallet.

Reply:
updated to `useAccountBalance`